src/app/util/external-api.js

```diff
@@ -2,6 +2,8 @@ const { put, post, get } = require('../dal/https-client');
 const { log } = require('./logger');
 const { AMOUNTS_MAGIC_NUMBERS, MERCHANT_DATA } = require('./static-data');
 
+const isValidResponseCode = (code) => code >= 200 && code < 300;
```

Comment:
If we move isValidResponseCode into https-client.js then we can reduce repeated code with this comparison?

Reply:
Probably should also be called isSuccessfulResponse code since  100-599 are all valid really